entity_plus.module

```diff
@@ -912,6 +949,10 @@ function entity_plus_theme() {
   $pattern = '(' . implode('|', $types) . ')(\.|__)';
 
   return array(
+		'entity_plus_status' => array(
```

Comment:
A few of the code sections have tabs at the start instead of spaces. Could you please fix those.

Reply:
My Bad there. I will make these updates and let you know asap. Thank you, I really appreciate it.